net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bfd_peers.xml

```diff
@@ -0,0 +1,231 @@
+			<url>pkg_edit.php?xml=frr/frr_ospf.xml</url>
+		</tab>
+		<tab>
+			<text>[OSPF6]</text>
+			<url>pkg_edit.php?xml=frr/frr_ospf6.xml</url>
+		</tab>
+		<tab>
+			<text>Status</text>
+			<url>/status_frr.php?protocol=bfd</url>
+		</tab>
+	</tabs>
+	<adddeleteeditpagefields>
+		<columnitem>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Shutdown</fielddescr>
+			<fieldname>shutdown</fieldname>
+		</columnitem>
+	</adddeleteeditpagefields>
+	<fields>		
+		<field>
+			<name>Peer Configurations</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+			<description>
+				<![CDATA[IP address of a peer]]>
+			</description>
+			<type>input</type>
+			<size>20</size>
+			<required/>
+		</field>
+		<field>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+			<description>
+				<![CDATA[
+				Label a peer with the provided word. This word can be referenced later on other daemons to refer to a specific peer.
+				]]>
+			</description>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Multihop</fielddescr>
+			<fieldname>multihop</fieldname>
+			<description>
+				<![CDATA[
+				 Tell the BFD daemon that we should expect packets with TTL less than 254 (because it will take more than one hop) and to listen on the multihop port (4784).
+				 When using multi-hop mode echo-mode will not work (see RFC 5883 section 3).
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
+		<field>
+			<fielddescr>Shutdown</fielddescr>
+			<fieldname>shutdown</fieldname>
+			<description>
+				<![CDATA[
+				Enables or disables the peer.
+				When the peer is disabled an ‘administrative down’ message is sent to the remote peer.
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
+
+		<field>
+			<name>Source Address/Interface</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Interface</fielddescr>
+			<fieldname>interface</fieldname>
+			<description>Select which interface we should use</description>
+			<type>select_source</type>
+			<source><![CDATA[frr_get_interfaces()]]></source>
+			<source_name>name</source_name>
+			<source_value>value</source_value>
+		</field>
+		<field>
+			<fielddescr>Local Source Address</fielddescr>
+			<fieldname>local_address</fieldname>
+			<description>
+				<![CDATA[
+				Provide a local address that we should bind our peer listener to and the address we should use to send the packets.<br />
+				This option is mandatory for IPv6.
```

Comment:
Reword to avoid "we".

Suggested: "Local IP address to which BFD will bind for sending and receiving packets to peers."